src/main/java/com/baidu/hugegraph/backend/store/tikv/TikvStdSessions.java

```diff
@@ -308,9 +308,7 @@ public void rollback() {
             int tablePrefixLength = startKey.size();
 
             List<Pair<byte[], byte[]>> result = regions.stream()
-                      .filter((region) -> {
-                             return !region.getStartKey().isEmpty();
-                       }).map((region) -> {
+                        .map((region) -> {
```

Comment:
can we replace the code with for-loop style? and then adjust the alignment

Reply:
ok